### PR TITLE
Feature/34

### PR DESCRIPTION
**Pull request by SuprunOleh**

### CODE REVIEW

File: app/src/main/java/com/github/frayeralex/bibiphelp/activities/ConfirmedHelpActivity.kt [hunk 1]
```diff
@@ -31,16 +31,23 @@ class ConfirmedHelpActivity : AppCompatActivity(), OnMapReadyCallback {
...
+    private var challenger: Int = 0
 
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
         setContentView(R.layout.activity_confirmed_help)
 
         eventId = intent.getStringExtra(IntentExtra.eventId)!!
 
+        challenger = intent.getIntExtra(IntentExtra.challenger, 0)
+
```
Comment (frayeralex) [inline app/src/main/java/com/github/frayeralex/bibiphelp/activities/ConfirmedHelpActivity.kt]: remove empty lines

File: app/src/main/java/com/github/frayeralex/bibiphelp/activities/ListEventActivity.kt [hunk 2]
```diff
@@ -89,6 +89,8 @@ class ListEventActivity : AppCompatActivity() {
         }
 
         fun refreshEvents(events: MutableList<EventModel>) {
+//            this.events =
```
Comment (frayeralex) [inline app/src/main/java/com/github/frayeralex/bibiphelp/activities/ListEventActivity.kt]: remove comment

File: app/src/main/java/com/github/frayeralex/bibiphelp/activities/MainActivity.kt [hunk 3]
```diff
@@ -263,6 +268,18 @@ class MainActivity : AppCompatActivity(), OnMapReadyCallback, GoogleMap.OnMarker
         return false
     }
 
+
+    private fun updateMapStyle() {
```
Comment (frayeralex) [inline app/src/main/java/com/github/frayeralex/bibiphelp/activities/MainActivity.kt]: remove this method - it is not used anymore

File: app/src/main/java/com/github/frayeralex/bibiphelp/activities/RejectHelpActivity.kt [hunk 4]
```diff
@@ -0,0 +1,132 @@
...
+import com.google.firebase.auth.FirebaseUser
+import kotlinx.android.synthetic.main.activity_details.*
+import kotlinx.android.synthetic.main.activity_reject_help.*
+
+class RejectHelpActivity: AppCompatActivity() {
+
+    companion object {
+        const val SAVIOR = 1
+    }
+
```
Comment (frayeralex) [inline app/src/main/java/com/github/frayeralex/bibiphelp/activities/RejectHelpActivity.kt:21]: remove empty lines

File: app/src/main/java/com/github/frayeralex/bibiphelp/activities/RejectHelpActivity.kt [hunk 5]
```diff
@@ -0,0 +1,132 @@
...
+            RequestStatuses.PENDING -> {
+                // todo add progress bar
+            }
+        }
+    }
+
+    private fun noRejectHelp () {
+        if (requestStatus == RequestStatuses.PENDING || user == null) return
+
+        viewModel.sendHelpRequest(eventId, user?.uid.toString())
```
Comment (frayeralex) [inline app/src/main/java/com/github/frayeralex/bibiphelp/activities/RejectHelpActivity.kt:64]: in this case, we should create a new ViewModel,
and remove userId from event.helpers Map

**Review by frayeralex — COMMENTED** [review]
need fix style code style issues

all logic fixes will be implemented in future tickets

File: app/src/main/java/com/github/frayeralex/bibiphelp/activities/ConfirmedHelpActivity.kt (hunk 1 above)
Comment (SuprunOleh) [reply app/src/main/java/com/github/frayeralex/bibiphelp/activities/ConfirmedHelpActivity.kt]: removed

File: app/src/main/java/com/github/frayeralex/bibiphelp/activities/ListEventActivity.kt (hunk 2 above)
Comment (SuprunOleh) [reply app/src/main/java/com/github/frayeralex/bibiphelp/activities/ListEventActivity.kt]: removed

File: app/src/main/java/com/github/frayeralex/bibiphelp/activities/MainActivity.kt (hunk 3 above)
Comment (SuprunOleh) [reply app/src/main/java/com/github/frayeralex/bibiphelp/activities/MainActivity.kt]: removed

File: app/src/main/java/com/github/frayeralex/bibiphelp/activities/RejectHelpActivity.kt (hunk 4 above)
Comment (SuprunOleh) [reply app/src/main/java/com/github/frayeralex/bibiphelp/activities/RejectHelpActivity.kt:21]: removed

File: app/src/main/java/com/github/frayeralex/bibiphelp/activities/RejectHelpActivity.kt (hunk 5 above)
Comment (SuprunOleh) [reply app/src/main/java/com/github/frayeralex/bibiphelp/activities/RejectHelpActivity.kt:64]: in next version)

**Review by frayeralex — APPROVED** [review]

**Pull request merged**